packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -96,8 +96,11 @@ const DateRangeCalendarMonthContainer = styled('div', {
 }));
 
 const DateRangeCalendarArrowSwitcher = styled(PickersArrowSwitcher)({
-  margin: CALENDAR_MARGIN,
-  marginBottom: CALENDAR_MARGIN / 2,
+  margin: `${CALENDAR_MARGIN}px 12px 8px 6px`,
+  '.MuiDateRangeCalendar-withoutWeekNumber &': {
```

Comment:
Can we import the class instead of hardcoding it ?

Reply:
Added an additional class and used it.